src/dvsim/flow/base.py

```diff
@@ -98,7 +101,7 @@ def __init__(self, flow_cfg_file, hjson_data, args, mk_config) -> None:
         self.rel_path = ""
         self.results_title = ""
         self.revision = ""
-        self.css_file = os.path.join(Path(os.path.realpath(__file__)).parent, "style.css")
+        self.css_file = Path(os.path.realpath(__file__)).parent / "style.css"
```

Comment:
```suggestion
        self.css_file = Path(__file__).resolve().parent / "style.css"
```
... to remove another use of os.path?

Reply:
Thanks!